packages/utilities/src/utils/getControlledValue.ts

```diff
@@ -8,7 +8,7 @@
 /**
  * Utility to determine controlled vs uncontrolled values
  */
-export function getControlledValue(...values: any[]) {
+export function getControlledValue<T>(...values: T[]): T | undefined {
```

Comment:
Just a reminder to merge and rebase this once #173 is merged to limit the scope of the breaking change in Lerna.